_studio/shared/include/mfx_config.h

```diff
@@ -123,6 +123,15 @@
     #endif
 #endif
 
+#if !defined(ANDROID)
+#define MFX_ENABLE_HEVCEHW_REFACTORING
```

Comment:
Why do we diverge for Android?

Reply:
it is related to some build issues. @eistarov could you comment here?

Reply:
Need to update Android.mk and fix couple of warnings - see PR draft #1858